core/src/apps/common/payment_request.py

```diff
@@ -13,6 +13,8 @@
     from apps.common.keychain import Keychain
 
 
+SLIP44_ID_RESERVED = const(0xFFFF_FFFF)
```

Comment:
nit: prefix with _ (i.e. `_SLIP44_ID_RESERVED`) to [enable inlining optimization](https://docs.micropython.org/en/latest/library/micropython.html#:~:text=On%20the%20other%20hand%2C%20if%20a%20constant%20begins%20with%20an%20underscore%20then%20it%20is%20hidden%2C%20it%20is%20not%20available%20as%20a%20global%20variable%2C%20and%20does%20not%20take%20up%20any%20memory%20during%20execution.).

Reply:
Shouldn't we rather call this SLIP44_ID_UNDEFINED or SLIP44_ID_NONE? Of course depends on what name we want to give it in SLIP-44, but we might as well have the discussion now. IMO "reserved" is way too vague about the meaning, basically saying "don't use this", in the future a meaning will be assigned to it. But we already know the meaning - undefined / not a coin.